mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -250,7 +258,10 @@ def get_shuttercount_normalised_correction_parameter(self) -> float:
         open_shuttercount = self.get_stack_shuttercounts(self._normalise_stack)
         if sample_shuttercount is None or open_shuttercount is None:
             return 1.0  # No shutter count data available so no correction needed
-        normalised_shuttercounts = sample_shuttercount / open_shuttercount
+        normalised_shuttercounts = np.divide(sample_shuttercount.astype(np.float32),
+                                             open_shuttercount.astype(np.float32),
+                                             out=np.ones_like(sample_shuttercount, dtype=np.float32),
+                                             where=open_shuttercount != 0)
```

Comment:
I noticed we actually needed to use a safe divide here to better handle zero values in ShutterCounts and remove warning from logs about invalid division.

Reply:
Or could just divide the first values in the array, and give an error if that is not a number.

Reply:
I want to divide the whole stacks as there's a chance we could still be required to use all available values to apply ShutterCount correction going forward when offering different levels of correction.  Happy to remove if we find that there's a low likelihood of the other values being useful for the correction.